src/gpu.cpp

```diff
@@ -66,7 +66,7 @@ GPUS::GPUS(overlay_params* params) : params(params) {
         std::shared_ptr<GPU> ptr = std::make_shared<GPU>(node_name, vendor_id, device_id, pci_dev);
         available_gpus.emplace_back(ptr);
 
-        SPDLOG_DEBUG("GPU Found: node_name: {}, vendor_id: {:x} device_id: {:x} pci_dev: {}", node_name, vendor_id, device_id, pci_dev);
```

Comment:
should be debug

Reply:
Shouldn't user be able to see this easier? Considering these things get printed once per launch. Instead of asking him to change his SPDLOG_LEVEL.

Reply:
The problem is that it's usually not once per launch, mangohud attaches to many random processes and spams the logs.
Unless something is out of the ordinary or something is wrong is when we want to print

Reply:
Oh it's been changed already, I'm learning I don't understand how this tool works.

Reply:
yeah, it's not really intuitive, i already force pushed all the changes